examples/pxScene2d/src/pxContextGL.cpp

```diff
@@ -1106,8 +1113,10 @@ static GLuint createShaderProgram(const char* vShaderTxt, const char* fShaderTxt
   program = glCreateProgram();
```

Comment:
Could assign `details.program` directly...

... equally 

`glAttachShader(program, details.fragShader);
glAttachShader(program, details.vertShader);`

... and avoid the local vars altogether.